metaci/cumulusci/models.py

```diff
@@ -8,15 +8,54 @@
 from cumulusci.core.exceptions import ScratchOrgException
 from django.core.cache import cache
 from django.db import models
+from django.conf import settings
 from django.urls import reverse
 from django.utils import timezone
+from django.core.exceptions import ValidationError
+
+from model_utils.managers import QueryManager
+
+import choices
 
 
 class Org(models.Model):
     name = models.CharField(max_length=255)
     json = models.TextField()
     scratch = models.BooleanField(default=False)
```

Comment:
`scratch` is redundant with `org_type`, should be removed.

Reply:
I agree. To avoid a super heavy refactor of every check for it, I'll add a scratch @property to the model